charts/geoweb-opmet-backend/templates/opmet-nginx-configmap.yaml

```diff
@@ -15,12 +15,12 @@ data:
 {{- if .Values.opmet.nginx.GEOWEB_REQUIRE_WRITE_PERMISSION }}
   GEOWEB_REQUIRE_WRITE_PERMISSION: {{ .Values.opmet.nginx.GEOWEB_REQUIRE_WRITE_PERMISSION | quote }}
 {{- end }}
-{{- if .Values.opmet.nginx.OPMET_BACKEND_HOST }}
-  OPMET_BACKEND_HOST: {{ .Values.opmet.nginx.OPMET_BACKEND_HOST | quote }}
+{{- if .Values.opmet.nginx.BACKEND_HOST }}
+  BACKEND_HOST: {{ .Values.opmet.nginx.BACKEND_HOST | quote }}
```

Comment:
These changed env var names should be updated in the readme file and also document the new var `opmet.nginx.registry | Registry to fetch nginx image'.

Reply:
I have updated the variable names in the readme on latest commit. Also sorted the variable table.